SpringProject/src/it/java/com/gradproject2019/conferences/ConferenceControllerIT.java

```diff
@@ -59,154 +59,120 @@ public void tearDown() {
 
     @Test
     public void shouldReturn200AndEmptyListWhenDatabaseEmpty() throws URISyntaxException {
-        //given
         URI uri = new URI(baseUrl);
 
-        //when
         ResponseEntity<List<ConferenceResponseDto>> response = getConferenceList(uri);
 
-        //Then
         Assert.assertEquals(200,response.getStatusCodeValue());
         Assert.assertEquals(true,response.getBody().isEmpty());
     }
 
     @Test
     public void shouldReturn200AndListOfConferencesWhenDatabasePopulated() throws URISyntaxException {
-        //given
         URI uri = new URI(baseUrl);
         Conference addedConference = conferenceRepository.saveAndFlush(conference);
 
-        //when
         ResponseEntity<List<ConferenceResponseDto>> response = getConferenceList(uri);
 
-        //Then
         Assert.assertEquals(200,response.getStatusCodeValue());
         Assert.assertEquals(conference.getName(),response.getBody().get(0).getName());
```

Comment:
Instead of doing .get(0) you can use extracting() and extract the fields you want to assert on from the ConferenceResponseDto